pkg/apiserver/model/application.go

```diff
@@ -24,19 +24,20 @@ import (
 )
 
 func init() {
-	RegistModel(&ApplicationComponent{}, &ApplicationPolicy{}, &Application{}, &ApplicationRevision{})
+	RegistModel(&ApplicationComponent{}, &ApplicationPolicy{}, &Application{}, &ApplicationRevision{}, &ApplicationWebhook{})
 }
 
 // Application application delivery model
 type Application struct {
 	Model
-	Name        string            `json:"name"`
-	Alias       string            `json:"alias"`
-	Project     string            `json:"project"`
-	Namespace   string            `json:"namespace"`
-	Description string            `json:"description"`
-	Icon        string            `json:"icon"`
-	Labels      map[string]string `json:"labels,omitempty"`
+	Name         string            `json:"name"`
+	Alias        string            `json:"alias"`
+	Project      string            `json:"project"`
+	Namespace    string            `json:"namespace"`
+	Description  string            `json:"description"`
+	Icon         string            `json:"icon"`
+	Labels       map[string]string `json:"labels,omitempty"`
+	WebhookToken string            `json:"webhookToken"`
```

Comment:
do we still need a field here?

Reply:
Or we need to get the token for every application in a LIST request.

Reply:
I think it can be deleted.